src/libmagickwand.jl

```diff
@@ -19,39 +19,131 @@ export MagickWand,
     writeimage
 
 # Find the library
-depsfile = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
-versionfile = joinpath(dirname(@__FILE__), "..", "deps", "versioninfo.jl")
-
+const depsfile = joinpath(dirname(@__DIR__), "deps", "deps.jl")
+const init_envs = Dict{String,String}()
 if isfile(depsfile)
     include(depsfile)
 else
-    error("ImageMagick not properly installed. Please run Pkg.build(\"ImageMagick\") then restart Julia.") # now that this is decoupled from images, should this be an error?
-end
-if isfile(versionfile)
-    include(versionfile)
-end
+    error("ImageMagick not properly installed. Please run Pkg.build(\"ImageMagick\") then restart Julia.")
+end
+
+const libmagic = Ref{Ptr{Void}}()
+func(fun::Symbol) = Libdl.dlsym(libmagic[], fun)
+
+const MagickWandGenesis                = Ref{Ptr{Void}}()
```

Comment:
Given the julia convention of using uppercase for types & modules, is it worth the extra verbosity of calling these `libwand_MagickWandGenesis`? I don't feel strongly about this, but thought I'd at least bring it up.